src/types/ScreenInfoUiaProviderBase.cpp

```diff
@@ -260,15 +253,19 @@ IFACEMETHODIMP ScreenInfoUiaProviderBase::GetRuntimeId(_Outptr_result_maybenull_
     *ppRuntimeId = nullptr;
 
     // AppendRuntimeId is a magic Number that tells UIAutomation to Append its own Runtime ID(From the HWND)
-    int rId[] = { UiaAppendRuntimeId, -1 };
+    std::array<int, 2> rId;

```

Comment:
we should be able to initializer list this one still.

Reply:
trying it.